test/Tinc/PackageSpec.hs

```diff
@@ -18,6 +18,11 @@ spec = do
         , Package "getopt-generics" "0.6.3"
         ]
 
+  describe "parseEmptyInstallPlan" $ do
```

Comment:
This should be a `context` of `parseInstallPlan` instead (see https://github.com/zalora/tinc/commit/ff08c48de3d354a0d4a1b20faa0c83ecd51a07a6).